blog/_posts/2016-07-08-RGSoC-2016-week-one.md

```diff
@@ -0,0 +1,23 @@
+---
+title: Week One of RGSoC
+layout: post
+created_at: Fri Jul 08 2016
+permalink: blog/2016-07-01-RGSoC-2016-day-one
+current: blog
+author: Ana Sofia
+twitter: RailsGirlsSoC
+categories:
+- news
+---
+
+![Our teams meet for the first time!](/img/blog/2016/2016-07-08-rgsoc-2016-day-one_online.png)<font color="grey"><small><i>Our teams meet for the first time! (Image: Ana Sofia Pinho with the help of the supervisors)</i></small></font>
+
+A week has passed and today we are in the middle of a [Global Kick-Off Party](http://railsgirlssummerofcode.org/blog/2016-06-29-kickoff-party)! Meanwhile, we thought you might be interested to see what all of our teams have been up in the first days of our program.
+
+From meeting all of the coaches, mentors and supervisors for the first time, making their first pull request or going to conferences, our students didn’t waste any time! We tried to select some of the best moments, but after reviewing all the pictures, we decided to make a facebook album. Take a look [here](https://www.facebook.com/permalink.php?story_fbid=1083849038362773&id=620914904656191)!
+
+![A week of RGSoC!](/img/blog/2016/2016-07-08-rgsoc-2016-day-one_tweets.png)<font color="grey"><small><i>Our teams having fun and getting some work done! :) (Image: Ana Sofia Pinho)</i></small></font>
+
+Oh! And if you want to be up to date with everything that’s going with the program and with our students, follow [us](https://twitter.com/RailsGirlsSoC) on twitter and our twitter [lists](https://twitter.com/RailsGirlsSoC/lists)! We created a bunch of lists for the teams, students, coaches, mentors and organisers. Just pick the ones you want!
+
+See you later at the Kick-Off Party? :)
```

Comment:
this sounds a little too "single-party-focused" so I will make a slight change to the phrasing.